docs/prysm-usage/get-more-peers.md

```diff
@@ -0,0 +1,59 @@
+---
+id: get-more-peers
+title: Get More Peers
+sidebar_label: Get more peers
+---
+
+ETH2 is designed to be a fully peer to peer (P2P) network.  This section describes how to configure Prysm and your network to optimise the number of peers that you communicate with on the ETH2 Network.  
+
+> NOTICE: This section contains advanced network configurations and is optional.
+
+ETH2 leverages [libp2p](../how-prysm-works/p2p-networking), a framework and suite of protocols for building peer-to-peer network applications.  When a beacon-chain node first starts up, it does two things to start communicating with other participants: 
+- It begins listening for new incoming p2p connections
```

Comment:
P2P is capitalised (except in the case of libp2p of course) 🙂

Reply:
fixed